Gemfile

```diff
@@ -176,6 +176,7 @@ gem 'unicorn', '~> 5.5', require: false
 gem 'god', '~> 0.13.7'
 
 group :staging, :production do
+  gem "alterity"
```

Comment:
Big :crossed_fingers: energy here. Not sure how to test that properly in close to real life conditions.

Reply:
We already have a pull request to add Departure instead (#4528), which I believe serves the same purpose, so it's probably a good idea to leave this out.

Reply:
Ok. Since this ticket is of a higher priority, I've added the `pt-online-schema-change` syntax to this PR so it isn't blocked until the gem is added.